tests/ui/test_other_user.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import os
+
+
+from ocs_ci.ocs.ui.base_ui import PageNavigator
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.utility.utils import get_ocp_version
+from ocs_ci.framework.testlib import ManageTest, ui
+from time import sleep
+from ocs_ci.utility.utils import exec_cmd
+
+logger = logging.getLogger(__name__)
+
+
+class OBCUI(PageNavigator):
+    def __init__(self, driver):
+        super().__init__(driver)
+        ocp_version = get_ocp_version()
+        self.sc_loc = locators[ocp_version]["OBC"]
+
+    def check_oc_option(self):
+        self.navigate_storageclasses_page()
+
+        "check OBC is visible to user after giving admin access"
+
+        self.do_click(self.sc_loc["create_obc"])
```

Comment:
What is the purpose? Only to press on create obc? If that so please add some verification that you are moved to the next page with checking the existence of some element in obc creation page.